docs: Command palette (#8808)
heartexlabs/label-studio: docs/source/guide/command_palette.md

---
title: Command palette
short: Command Palette
tier: enterprise
type: guide
order: 0
order_enterprise: 356
meta_title: Enhanced search with the command palette
meta_description: How to use the Label Studio  command palette
section: "Manage Your Organization"
date: 2025-01-28 16:40:16
---

The command palette is an enhanced search and command tool. 

> To open it, click the search icon in the upper right or press `Ctrl+K`/`Cmd+K`

Reviewer: ```suggestion
To open it, click the search icon in the upper right or press `Ctrl+K` (`Cmd+K` on Mac)
```

![Screenshot](/images/admin/command-palette1.png)

## What you can do

![Screenshot](/images/admin/command-palette2.png)

From here you can:

- Search across your Label Studio org. For example, you can search for projects and workspaces.
- Navigate to pages within the app. 
- Find quick links to Label Studio documentation and resources, such as the API docs and the changelog. 

> - When you are in a project, you can quickly jump to settings pages or individual settings. 

Reviewer: ```suggestion
- When you are in a project, you can search for settings pages or individual settings to quickly jump to them. 
```

## Role-aware results 

Results respect your role and access. For example, Annotators and Reviewers won’t see workspaces or projects that they can’t access.

## AI assistant

If AI assistant is enabled, you can enter a query and then press Tab. The assistant opens automatically with your query.

## Keyboard shortcuts

> - `Ctrl+K` or `Cmd+K` to open the command palette 

Reviewer: ```suggestion
- `Ctrl+K` (`Cmd+K` on Mac) to open the command palette 
```

> - Use the arrow keys to navigate through the results

Reviewer: ```suggestion
- Use the `Up` and `Down` arrow keys to navigate through the results
```

> - Enter to select an option

Reviewer: ```suggestion
- `Enter` to select an option
```

> - Esc to close

Reviewer: ```suggestion
- `Esc` to close
```
